{"name": "Create user where one of the fields has invalid data", "status": "passed", "parameters": [{"name": "user_data", "value": "{'email': '', 'password': 'yandex*', 'name': 'Sam'}"}], "start": 1727745325170, "stop": 1727745325553, "uuid": "24b314a4-65f3-4001-8f0b-6c248f799204", "historyId": "8e2c4ba70fa87a021af9d0927957f4a1", "testCaseId": "097b98d45b92e290e2b047ca8ae14df0", "fullName": "tests.test_create_user.TestCreateUser#test_create_user_with_invalid_data", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_user"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "mac.lan"}, {"name": "thread", "value": "31983-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_user"}]}